php/WP_Mock.php

```diff
@@ -1,40 +1,13 @@
 <?php
-/**
- * WP_Mock
- *
- * LICENSE
- *
- * Copyright 2013 10up and other contributors
- * http://10up.com
- *
- * Permission is hereby granted, free of charge, to any person obtaining
- * a copy of this software and associated documentation files (the
- * "Software"), to deal in the Software without restriction, including
- * without limitation the rights to use, copy, modify, merge, publish,
- * distribute, sublicense, and/or sell copies of the Software, and to
- * permit persons to whom the Software is furnished to do so, subject to
- * the following conditions:
- *
- * The above copyright notice and this permission notice shall be
- * included in all copies or substantial portions of the Software.
- *
- * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
- * EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
- * MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
- * NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE
- * LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
- * OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
- * WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
- *
- * @package    WP_Mock
- * @copyright  Copyright (c) 2013 10up (http://10up.com)
- * @license    MIT License
```

Comment:
These rows may still be worthwhile maintaining in this file, with an update from MIT to BSD-3-Clause to make it explicit in the main file what the licensing is?

Reply:
sure thing!

Reply:
how about [`2160a9b` (#218)](https://github.com/10up/wp_mock/pull/218/commits/2160a9b092a75808dcd2bcc5a579ac0faeb9cae1)